torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -267,6 +267,14 @@ c10::optional<Node*> tryMerge(
       getHeader(consumer),
       ":\n");
 
+  // Shortcut for Block code generation for single op
+  if (tensorexpr::getTEGenerateBlockCode() &&
+      producer->kind() == prim::Constant &&
+      (consumer->kind() == aten::add || consumer->kind() == aten::mul)) {
```

Comment:
Minor nitpick: I expect this list to get fairly long. So it is better to separate into its own function, and follow similar structure here to make the list easier to view. 

https://github.com/pytorch/pytorch/blob/6bdfd6ae1adb341f5bae2fc8bc987653b8a319ad/torch/csrc/jit/passes/tensorexpr_fuser.cpp#L20